src/test/resources/features/Returns.feature

```diff
@@ -46,4 +46,123 @@ Feature: Submit ECL Return
+    Then I should see an error that says <Expected content>
+    Examples:
+      | Days  | Expected content                      |
+      |       | Enter the length                      |
+      | 0     | Length must be between 1 and 999 days |
+      | -1    | Length must be between 1 and 999 days |
+      | 1000  | Length must be between 1 and 999 days |
+      | asdwa | Length must be a whole number         |
+      | 245.0 | Length must be a whole number         |
+
+  Scenario Outline: User enters an invalid amount for UK revenue for their relevant accounting period
+    Given I am signed in to the return journey
+    When I enter the UK revenue <UK Revenue> for my relevant accounting period
+    Then I should see an error that says <Expected content>
+    Examples:
+      | UK Revenue | Expected content                                              |
+      |            | Enter the UK revenue                                          |
+      | -1         | UK revenue must be between 0 and 99,999,999,999               |
+      | asdfg      | UK revenue must be a number rounded down to the nearest pound |
+      | 99.999     | UK revenue must be a number rounded down to the nearest pound |
+      | 1019999.99 | UK revenue must be a number rounded down to the nearest pound |
+
+  Scenario: User does not select whether or not they carry out AML-regulated activity for the full financial year?
+    Given I am signed in to the return journey
+    When I do not select an option for whether or not I carry out AML-regulated activity for the full financial year?
+    Then I should see an error that says Select if you carried out AML-regulated activity in the financial year
+
+  Scenario: User select no whether or not they carry out AML-regulated activity for the full financial year?
+    Given I am signed in to the return journey
+    When I select No option for my AML-regulated activity for the full financial year
+    Then I should see an error that says How many days of the financial year did you carry out AML-regulated activity?
+
+  Scenario Outline: User enters an invalid number of days of AML regulated activity in the financial year
+    Given I am signed in to the return journey
+    When I enter the total number of days <Days> I carried out AML regulated activity
+    Then I should see an error that says <Expected content>
+    Examples:
+      | Days  | Expected content                                                |
+      |       | Enter the number of days you carried out AML-regulated activity |
+      | 0     | Length must be between 1 and 999 days                           |
+      | -1    | Length must be between 1 and 999 days                           |
+      | 1000  | Length must be between 1 and 999 days                           |
+      | asdwa | Length must be a whole number                                   |
+      | 245.0 | Length must be a whole number                                   |
```

Comment:
This should be 365, not 999 - I've raised separately with @TonyCOgilvie

Reply:
leave this for now - Tony is fixing with the current work he is doing, so we can update the tests after that